src/me/StevenLawson/TotalFreedomMod/Listener/TFM_BlockListener.java

```diff
@@ -111,6 +112,14 @@ else if (playerLocation.distanceSquared(location) > (nukeMonitorRange * nukeMoni
                 }
             }
         }
+        if (player.getGameMode().equals(GameMode.SURVIVAL))
```

Comment:
Change this to:

`         if (player.getGameMode().equals(GameMode.SURVIVAL))
         {
          if (TFM_ConfigEntry.AUTO_ENTITY_WIPE.getBoolean()) 
             {
                 if (TFM_ConfigEntry.AUTO_ENTITY_WIPE.equals(false)) 
                 {
                     return;
                 }
                 else
                 {
             for (ItemStack stack : event.getBlock().getDrops())
             {
                 event.getBlock();
                 player.getInventory().addItem(stack);
             }
            }
                 
             }
         }`